standard/statements.md

```diff
@@ -1961,6 +1981,22 @@ Otherwise, the expansion is
 }
 ```
 
+For ref struct resources, the only possible expansion is
+
+```csharp
+{
+    ResourceType resource = «expression»;
```

Comment:
I wonder whether `ResourceType` ought to be in chevrons here and above? (It's a bit of a third case - it's not "something from the original code" nor is it "fixed text"...)